app/views/admin/invites/new.html.haml

```diff
@@ -0,0 +1,29 @@
+.row
+  .col-md-offset-1.col-md-8
+    %h1.text-center Late Submission Invitation
+    = semantic_form_for(@invite, url: admin_conference_invites_path, html: { multipart: true }) do |f|
+      = f.input :emails, class: 'form-control', required: true,
+        hint: 'Add emails to invite people to join the app and submit requests after the cfps deadline.'
+      = f.input :invite_for, as: :select, collection: ['Track', (t 'booth').capitalize]
+      = f.input :end_date, as: :string, input_html: { id: 'invitation-end-date', end_date: @conference.end_date }
```

Comment:
Perhaps this should be readonly so that I cannot press backspace/delete button and remove the text?

Reply:
Yes